README.md

```diff
@@ -3677,6 +3677,39 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
     </details>
 
+* <a id='generic-void'></a>(<a href='#generic-void'>link</a>) **Avoid using `()` as a generic type**. Prefer `Void`.
```

Comment:
Instead of saying "Avoid using `()` as a generic type" maybe we should say "Avoid using `()` as a _type_"

e.g. the rule would convert `let result: ()` to `let result: Void` (right?) but that's not covered by the "generic type" wording here

Reply:
Good point, let me update it